### PR TITLE
Add Groovy Hello World

### PR DESCRIPTION
**Pull request by Christian-Oleson**
Also, addressed a few minor issues in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,25 +24,25 @@ Hacktoberfest is for everyone. Whether it’s your first time—or your ninth—
 # **Steps to Follow!!**
 
 ### **1. Register for Hacktoberfest**
-Follow the link and authorize with either Github or Gitlab to participate in Hacktoberfest! <br>
+Follow the link and authorize with either GitHub or Gitlab to participate in Hacktoberfest! <br>
```
Comment (Christian-Oleson) [inline README.md:27]: Invalid Github name

File: Kotlin/hello-world.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// This is the main function
```
Comment (Christian-Oleson) [inline Kotlin/hello-world.kt:1]: As a side note, I did not see in the Java file instructions to a newbie on how to compile and run a java class. This isn't the end of the world, but overall someone should consider adding documentation or links on how to do this.

**Comment by reveurguy** [comment]
Hi @Christian-Oleson , thanks for making the Readme changes. Also thanks for you effort but a hello world program in kotlin has been accepted already. you can try out any new language!

File: Kotlin/hello-world.kt (hunk 2 above)
Comment (reveurguy) [reply Kotlin/hello-world.kt:1]: Yeah, thanks for the suggestion but given that the repository focuses on hello world programs in as many languages as possible, it is not practically possible to add instructions for every language. You can always search online for help for installation or any other issue as there are a great amount of resources available for that.

**Comment by reveurguy** [comment]
If you can remove the kotlin code, i'll merge this PR for the readme changes. For the program you can make a new PR adding a program in any other language that already hasn't been used.

**Review by reveurguy — CHANGES_REQUESTED** [review]
remove the kotlin program folder and file.

**Comment by Christian-Oleson** [comment]
> If you can remove the kotlin code, i'll merge this PR for the readme changes. For the program you can make a new PR adding a program in any other language that already hasn't been used.

To add a little more benefit versus just some changes to the readme, I also added the groovy hello world.

**Pull request merged**